src/functionBased/components/InputTodo.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { FaPlusCircle } from 'react-icons/fa';
+import PropTypes from 'prop-types';
+
+const InputTodo = (props) => {
+  const [inputText, setInputText] = useState({
+    title: '',
+  });
+
+  const onChange = (e) => {
+    setInputText({
+      ...inputText,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const { addTodoProps } = props;
+    if (inputText.title.trim()) {
+      addTodoProps(inputText.title);
+      setInputText({
+        title: '',
+      });
+    } else {
+      alert('Please write item');
+    }
```

Comment:
As per the JavaScript best practices, I suggest you avoid the use of alert() in your code. You can use an inline element like a span tag to display your message to your user